source/Scripts/CompileRootConfiguration.ps1

```diff
@@ -12,6 +12,8 @@ if (-not $environment)
     $environment = 'NA'
 }
 
+$rsopCache = Get-DatumRsopCache
```

Comment:
This has to be explained.
What are you doing, why.
Basically, doing the RSOP has a high cost.